### PR TITLE
To fix ontology events always being made.

### PR DESCRIPTION
**Pull request by jkbaumohl**

### CODE REVIEW

File: lib/GenomeFileUtil/core/GenbankToGenome.py [hunk 1]
```diff
@@ -639,22 +624,49 @@ def _get_seq(self, feat, contig):
                                [part.start:part.end].reverse_complement()))
         return "".join(seq)
 
+    def _create_ontology_event(self,ontology_type):
```
Comment (JamesJeffryes) [inline lib/GenomeFileUtil/core/GenbankToGenome.py]: Can we move this and the identical version in GFF uploader into the utils so they are not duplicated? (If we chose to support a new ontology, only want to have to make a change in one place)

**Comment by jkbaumohl** [comment]
If in utils will it recognize "self." ?

**Comment by JamesJeffryes** [comment]
No, This would require some rewrite. If you can confirm the unit tests are passing I'll merge

**Comment by jkbaumohl** [comment]
Hey James,

Yes I confirmed the standard and GB unit tests ran fine.

-Jason

**Pull request merged**